lib/agenda.js

```diff
@@ -57,8 +57,8 @@ WHERE {
     ^besluitvorming:vindtPlaatsTijdens ?agendaActivity .
 
   ?agendaActivity besluitvorming:genereertAgendapunt ?agendaitem ;
-    mu:uuid ?agendaActivityId ;
-    dossier:startDatum ?agendaActivityStart .
+    mu:uuid ?agendaActivityId .
+  OPTIONAL { ?agendaActivity dossier:startDatum ?agendaActivityStart . }
```

Comment:
`agendaActivity` date was inserted in this migration
https://github.com/kanselarij-vlaanderen/app-kaleidos/blob/development/config/migrations/20200629150006-add-startdate-to-activities.sparql#L6

But the date was dependent on the existence of the date of agendaitem `besluitvorming:aanmaakdatum` (which is now migrated to `dct:created`)

We can't determine when an agendaitem was "created" in legacy, so no date makes sense.

1. should we look at legacy for this again? do we even have agendaActivities for all agendaitems since we didn't make the date optional?
2. if no to the first, optional date should be fine as long as the frontends `getRelatedAgendas` can handle a null.